--- openvcpd-0.3-rc2/configure.ac~	2007-09-04 21:22:02.000000000 +0200
+++ openvcpd-0.3-rc2/configure.ac	2007-09-22 01:07:52.000000000 +0200
@@ -9,13 +9,13 @@
 AC_PROG_CC
 
 # Checks for libraries.
+AC_CHECK_LIB([dl], [dlopen])
 AC_CHECK_LIB([vserver], [vc_ctx_migrate], ,[AC_MSG_ERROR([libvserver(util-vserver) not found])])
 AC_CHECK_LIB([sqlite3], [sqlite3_close], ,[AC_MSG_ERROR([libsqlite3 not found])])
 AC_CHECK_LIB([xml2], [xmlTextReaderRead], ,[AC_MSG_ERROR([libxml2 not found])])
 AC_CHECK_LIB([pcap], [pcap_loop], ,[AC_MSG_ERROR([libpcap not found])])
 AC_CHECK_LIB([iptc], [iptc_init], ,[AC_MSG_ERROR([libiptc (iptables-dev) not found])])
 AC_CHECK_LIB([iptables], [do_command], ,[AC_MSG_ERROR([libiptables (iptables-dev) not found])])
-AC_CHECK_LIB([dl], [dlopen])
 
 AC_ARG_WITH([gnutls],
         [AS_HELP_STRING([--with-gnutls],
@@ -69,7 +69,7 @@
 fi
 
 if test "x$with_ipv6" != xno; then
-        CFLAGS+=" -DIPV6_SUPPORT"
+        CFLAGS="$CFLAGS -DIPV6_SUPPORT"
 fi
 
 xmlflags=`pkg-config libxml-2.0 --cflags`
@@ -77,7 +77,7 @@
 	AC_MSG_ERROR([libxml2 headers not found])
 fi
 
-CFLAGS+=" $xmlflags"
+CFLAGS="$CFLAGS $xmlflags"
 
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
--- openvcpd-0.3-rc2/Makefile.in~	2007-09-22 22:32:36.879983779 +0200
+++ openvcpd-0.3-rc2/Makefile.in	2007-09-22 22:37:31.549731799 +0200
@@ -2,10 +2,11 @@
 bindir = @bindir@
 datadir = @datadir@
 sysconfdir = @sysconfdir@
+sharedstatedir = @sharedstatedir@
 iptablever=`iptables -V |  awk  '{ print $$2 }' | cut -c2-`
 
 CC=@CC@
-CFLAGS=@CFLAGS@ @DEFS@ -DOPENVCP_CONFIG=\"$(sysconfdir)/openvcpd.conf\" -DOPENVCP_DATA=\"$(datadir)/openvcpd\" -DIPTABLES_VERSION=\"$(iptablever)\" -Ilib/ -Isrc/ -rdynamic
+CFLAGS=@CFLAGS@ @DEFS@ -DOPENVCP_CONFIG=\"$(sysconfdir)/openvcpd.conf\" -DOPENVCP_DATA=\"$(sharedstatedir)/openvcpd\" -DIPTABLES_VERSION=\"$(iptablever)\" -Ilib/ -Isrc/ -rdynamic
 LDFLAGS=@LDFLAGS@
 LIBS=@LIBS@
 
