--- openvcpd-0.4rc2/configure.ac.orig	2009-02-07 20:04:17.000000000 +0100
+++ openvcpd-0.4rc2/configure.ac	2009-05-03 00:33:58.000000000 +0200
@@ -12,12 +12,12 @@
 AC_PATH_PROG([PKGCONFIG], [pkg-config], [no])
 
 # Checks for libraries.
+AC_CHECK_LIB([dl], [dlopen])
 AC_CHECK_LIB([vserver], [vc_ctx_migrate], ,[AC_MSG_ERROR([libvserver(util-vserver) not found])])
 AC_CHECK_LIB([sqlite3], [sqlite3_close], ,[AC_MSG_ERROR([libsqlite3 not found])])
 AC_CHECK_LIB([xml2], [xmlTextReaderRead], ,[AC_MSG_ERROR([libxml2 not found])])
 AC_CHECK_LIB([pcap], [pcap_loop], ,[AC_MSG_ERROR([libpcap not found])])
 AC_CHECK_LIB([iptc], [iptc_init], ,[AC_MSG_ERROR([libiptc (iptables-dev) not found])]) 
-AC_CHECK_LIB([dl], [dlopen])
 
 AC_ARG_WITH([gnutls],
         [AS_HELP_STRING([--with-gnutls],
--- openvcpd-0.4rc2/Makefile.in.orig	2009-02-07 20:04:17.000000000 +0100
+++ openvcpd-0.4rc2/Makefile.in	2009-05-03 00:35:59.000000000 +0200
@@ -2,10 +2,11 @@
 bindir = @bindir@
 datadir = @datarootdir@
 sysconfdir = @sysconfdir@
+sharedstatedir = @sharedstatedir@
 
 REV:=$(shell svnversion 2>/dev/null | sed 's/M//')
 CC=@CC@
-CFLAGS=@CFLAGS@ @DEFS@ -DOPENVCP_CONFIG=\"$(sysconfdir)/openvcpd.conf\" -DOPENVCP_DATA=\"$(datadir)/openvcpd\" -Ilib/ -Isrc/ -rdynamic -DREVISION=\"$(REV)\"
+CFLAGS=@CFLAGS@ @DEFS@ -DOPENVCP_CONFIG=\"$(sysconfdir)/openvcpd.conf\" -DOPENVCP_DATA=\"$(sharedstatedir)/openvcpd\" -Ilib/ -Isrc/ -rdynamic -DREVISION=\"$(REV)\"
 LDFLAGS=@LDFLAGS@
 LIBS=@LIBS@
 
